spec/services/expression_evaluators/retail_price_spec.rb

```diff
@@ -1,9 +1,18 @@
 RSpec.describe ExpressionEvaluators::RetailPrice, :user_session do
   subject(:evaluator) { described_class.new(measure, measure_component) }
 
-  let(:measure) { build(:measure, :excise) }
-  let(:measure_component) { build(:measure_component, :with_retail_price_measure_units) }
-  let(:user_session) { build(:user_session, :with_retail_price_measure_amount, commodity_code: '0103921100') }
+  include_context 'with a fake commodity'
+
+  let(:user_session) { build(:user_session, :with_retail_price_measure_amount, commodity_code: commodity.code) }
+
+  let(:measure) { commodity.import_measures.first }
+  let(:measure_component) { measure.measure_components.first }
+
+  let(:commodity) do
+    measure = attributes_for(:measure, :excise, measure_components: [attributes_for(:measure_component, :with_retail_price_measure_units)])
```

Comment:
this assignment could be problematic, 
we have already a let names `measure`,
do you want to use the `let` or the `var` ?